opwen_email_server/utils/email_parser.py

```diff
@@ -80,3 +83,28 @@ def _get_recipients(email: dict) -> Iterable[str]:
 def get_domains(email: dict) -> Iterable[str]:
     return frozenset(address.split('@')[-1]
                      for address in _get_recipients(email))
+
+
+def _get_as_base64(image_url):
+    res = requests.get(image_url)
+
+    if res.status_code != 200:
+        res.raise_for_status()
+
+    img_type = res.headers['Content-Type']
+    img_content = b64encode(res.content).decode('ascii')
+    base64 = "data:" + img_type + ";base64," + img_content
```

Comment:
Nit: this might be cleaner using string formatting: `base64 = 'data:{};base64,{}'.format(img_type, img_content)`.